src/RequestHandlerSwooleRunner.php

```diff
@@ -30,6 +32,19 @@
  */
 class RequestHandlerSwooleRunner extends RequestHandlerRunner
 {
+    /**
+     * Default static file extensions supported
+     */
+    const DEFAULT_STATIC_EXTS = [
+        'css'  => 'text/css',
```

Comment:
@ezimuel Is html missing here or was skip on purpose?

Reply:
I forgot it. I need to update the default list.